src/defs.h

```diff
@@ -308,6 +310,8 @@ struct insn {
     var_t *rs1;
     var_t *rs2;
     int sz;
+    bool useful; /* Used in DCE process. Set true if instruction is useful. */
+    basic_block_t *belong_to_bb;
```

Comment:
How about naming it `belong_to`?

Reply:
No problem, I will make the change. I originally wanted to distinguish it from the `belong_to` within the `basic_block_t`.